src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -2462,6 +2469,51 @@ def update_nsg_rule_2017_03_01(instance, protocol=None, source_address_prefix=No
 # endregion
 
 
+# region NetworkProfiles
+def create_network_profile(cmd, resource_group_name, network_profile_name, location=None, tags=None):
+    client = network_client_factory(cmd.cli_ctx).network_profiles
+    NetworkProfile = cmd.get_models('NetworkProfile')
+    profile = NetworkProfile(location=location, tags=tags)
+    return client.create_or_update(resource_group_name, network_profile_name, profile)
+
+
+def update_network_profile(instance, tags=None):
+    if tags is not None:
+        instance.tags = tags
+    return instance
+
+
+def list_network_profiles(cmd, resource_group_name=None):
+    client = network_client_factory(cmd.cli_ctx).network_profiles
+    if resource_group_name:
+        return client.list(resource_group_name)
+    return client.list_all()
+
+
+def create_np_container_nic_config(cmd, resource_group_name, network_profile_name, nic_config_name):
```

Comment:
Are this method and the below one relevant to NRP profile support?

Reply:
No I will remove them.  They are part of why there is no create/update commands